can_tools/scrapers/base.py

```diff
@@ -91,7 +92,7 @@ def quit_early(self) -> bool:
 
     def _retrieve_dt(self, tz: str = "US/Eastern") -> pd.Timestamp:
         """Get the current datetime in a specific timezone"""
-        out = pd.Timestamp.utcnow().tz_convert(tz).normalize().tz_localize(None)
+        out = self.execution_dt.tz_convert(tz).normalize().tz_localize(None)
```

Comment:
I updated this as part of fixing the database. For the scrapers that use the `_retrieve_dt` to get the information, using the current date doesn't make sense for when they are re-run...

Same logic for `_retrieve_vintage`